flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -2198,15 +2196,8 @@ var A320_Neo_UpperECAM;
             return "%";
         }
         getValue(_engine) {
-            const imbalance = SimVar.GetSimVarValue("L:A32NX_ENGINE_IMBALANCE", "number");
-            let n2Imbalance = 0;
-            if (parseInt(imbalance.toString().substr(0, 1)) == _engine) {
-                n2Imbalance = parseFloat(imbalance.toString().substr(5, 2)) / 100;
-            } else {
-                n2Imbalance = parseFloat(0);
-            }
-            const name = "ENG N2 RPM:" + _engine;
-            let percent = SimVar.GetSimVarValue(name, "percent") - n2Imbalance;
+            const name = "L:A32NX_ENGINE_N2:" + _engine;
```

Comment:
creation of name can be done inline like for N1

Reply:
Change committed.